datadreamer/pipelines/generate_dataset_from_scratch.py

```diff
@@ -104,11 +105,18 @@ def parse_args():
         help="Image generator to use",
     )
     parser.add_argument(
-        "--image_annotator",
```

Comment:
Shouldn't we create a deprecation warning for the arguments that are no longer used?

Reply:
Yes, we should. Thanks!

Or, if you have another idea for the annotator flag names, I'm open to them. @sokovninn

Reply:
Ok, an alternative option would be to keep the current name `--image_annotator` and check if `--task` is compatible with the specified annotator, which we should do in any case. But not sure which is better.

Reply:
I changed it back to using `--image_annotator` flag and added the checks.